### DISCUSSION TITLE
Hiding columns by default

### DISCUSSION THREAD

**Discussion by nckre** [sadmann7/tablecn#783] [Ideas]
Hey, given that there `enableHiding`, would be cool if there was an accompanying setting `initialHidden` to auto-hide certain columns for a cleaner first load (e.g. usually don't want to show createdAt but sometimes it's useful to have it)

**Comment by nckre** [comment]
There is `columnVisbility` in `initialState` that does this

**Reply by noxify** [reply]
For everyone else who has this problem.


Inside the `useDataTable` hook, you can add the following

```
const { table } = useDataTable({
  // ... other table config
  defaultVisibility: { createdAt: false },
})
```
